### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [ArchiveBox/ArchiveBox#1067] [Q&A] [answered]
Hi all.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by ntevenhere** [comment, marked as answer]
How can you help?

I'm not ArchiveBox's spokesperson or a main dev but I can give you the robotic answer: Regarding the project's activity see the commit log, and the ambitious [Roadmap](https://github.com/ArchiveBox/ArchiveBox/wiki/Roadmap), looks like we're nowhere near done :)

**Reply by ghost** [reply]
Hi notevenaperson.

> How can you help?
> I'm not ArchiveBox's spokesperson or a main dev but I can give you the robotic answer: Regarding the project's activity see the commit log, and the ambitious [Roadmap](https://github.com/ArchiveBox/ArchiveBox/wiki/Roadmap), looks like we're nowhere near done :)

thank you for feedback.

**Comment by pirate** [comment]
It's active but has been slow over the last year because I've had limited time/energy to work on it with a recent big move across country and many life changes.

It's still my main side project and I intend to keep working on it and reviewing/merging/pushing my own PRs. That being said my pace and commitment level is sporadic depending on my energy for side projects.

If you're working on a commercial project and are considering using ArchiveBox for something critical or customer-facing, I'd recommend checking out a better funded alternative like browsertrix-crawler instead (developed by a friend of mine, it's great): https://github.com/ArchiveBox/ArchiveBox/wiki/Web-Archiving-Community#other-archivebox-alternatives

**Reply by ghost** [reply]
Hi pirate.

> It's active but has been slow over the last year because I've had limited time/energy to work on it with a recent big move across country and many life changes.
> It's still my main side project and I intend to keep working on it and reviewing/merging/pushing my own PRs. That being said my pace and commitment level is sporadic depending on my energy for side projects.
> If you're working on a commercial project and are considering using ArchiveBox for something critical or customer-facing, I'd recommend checking out a better funded alternative like browsertrix-crawler instead (developed by a friend of mine, it's great): https://github.com/ArchiveBox/ArchiveBox/wiki/Web-Archiving-Community#other-archivebox-alternatives

thank you for feedback.
